packages/utils/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
-    "lib": ["es2020"],
-    "target": "es2020",
+    "lib": ["es5", "es6", "es2019"],
+    "target": "es5",
```

Comment:
question: We can't have this bumped to es2020?

Reply:
Yeap, until the core isn't using es2020 we can't change it here :/